src/app/content/courses/create-your-sdk-with-codama/codama-with-anchor/en.mdx

```diff
@@ -0,0 +1,180 @@
+import { ArticleSection } from "../../../../components/ArticleSection/ArticleSection";
+
+# Build your Codama IDL from Anchor IDL
+
+Creating a Codama IDL from Anchor is extremely straightforward. Once you generate your Anchor IDL with anchor build, you can convert it directly using the `createFromRoot` function:
```

Comment:
`rootNodeFromAnchor` is the function that does the conversion, `createFromRoot` just wraps the `RootNode` in a `Codama` class for a smoother developer experience when visiting nodes.

```suggestion
Creating a Codama IDL from Anchor is extremely straightforward. Once you generate your Anchor IDL with anchor build, you can convert it directly using the `rootNodeFromAnchor ` function:
```